core/jvm/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.8
```

Comment:
I don't think you want to change this.

Reply:
I don't recall changing that *sigh* I've had so many issues with intellij today I've done almost nothing useful. Might be time to start using something else.

Reply:
SBT version is 1.3.3 on `master` so don't think we want to change to an older SBT version or really do anything with it at all as part of this PR, right?

Reply:
Sorry, didn't see your response so wasn't sure if you were confused about my comment or what happened.

Reply:
I'll change it back. I have no idea how it even changed since I don't believe I've even opened that file. Still I should have been checking the commit screen more closely. Another lesson learnt.